styles/style.css

```diff
@@ -291,7 +291,7 @@ article h4 {
 }
 
 .main-program li i {
-  width: 45px;
+  width: 2.8rem;
 }
```

Comment:
- Good job working on the project so far. Kindly try to match the Figma design as much as possible, which includes:
   - [ ] Removing the Sponsors section on the About page to match the Figma design.
   - [ ] Changing the background of the footer in the About page to match the Figma design.
 
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/163216040-30af91a9-79e3-440c-be24-7c0260a48d00.png) | ![image](https://user-images.githubusercontent.com/62290545/163215855-78de6531-2e21-4c17-9b8a-d52fac396889.png) |

Reply:
All good now 👍🏽